turbinia/evidence.py

```diff
@@ -282,12 +304,31 @@ def __init__(self, project=None, zone=None, disk_name=None, *args, **kwargs):
     super(GoogleCloudDisk, self).__init__(*args, **kwargs)
     self.cloud_only = True
 
+    self._attached_path = None
+
   def _preprocess(self):
-    self.local_path = google_cloud.PreprocessAttachDisk(self.disk_name)
+    self._attached_path = google_cloud.PreprocessAttachDisk(self.disk_name)
+    self.loopdevice_path = mount_local.PreprocessLosetup(self._attached_path)
```

Comment:
And maybe for this one we could use volume_path (or partition_path) instead of loopdevice_path?  As per our earlier conversation I'm just trying to find common attribute names across evidence types.

Reply:
this is going to be /dev/loopX  and not /dev/loopXpY
the mount preprocessor takes  /dev/loopX path and a partition number to build the path to /dev/loopXpY.
A TODO could be to have evidence.GetVolumePath(volume_number) that does this

device_path =  google_cloud.PreprocessAttachDisk
volume_path = self.GetVolumePath(device_path, volume_number) // or partition_number
mount_path = mount_local.mount(volume_path)